src/Orleans.Core.Abstractions/Statistics/IHostEnvironmentStatistics.cs

```diff
@@ -2,10 +2,30 @@ namespace Orleans.Statistics
 {
     public interface IHostEnvironmentStatistics
     {
+        /// <summary>
+        /// Total physical memory on the host in bytes
+        /// i.e. 16426476000L for 16 gb
+        /// </summary>
+        /// <value>16426476000</value>
         long? TotalPhysicalMemory { get; }
 
+        /// <summary>
+        /// Host CPU usage from 0.0-1.0
+        /// i.e. 0.7f for 70% CPU usage
+        /// </summary>
+        /// <value>0.7</value>
         float? CpuUsage { get; }
 
+        /// <summary>
+        /// Total memory available for allocation on the host in bytes
+        /// i.e. 14426476000L for 14 gb
+        /// </summary>
+        /// <value>14426476000</value>
         long? AvailableMemory { get; }
+
+        /// <summary>
+        /// Total memory usage in bytes as reported from <code>GC.GetTotalMemory(false)</code>
+        /// </summary>
+        long MemoryUsage { get; }
```

Comment:
So I added this to the interface, but we could probably make naming more obvious here since the other three properties refer to the  host, while this property refers only to the process

Reply:
We already have this value in `IAppEnvironmentStatistics.MemoryUsage` (implemented in `AppEnvironmentStatistics`). Since this logic in cross plat, I don't think we should add it here.

Yes, there is a `MemoryUsage` field in `PerfCounterEnvironmentStatistics` but it's not used anywhere in Orleans (as far as I know), I guess we kept it to avoid a breaking change.